tests/utils/test_dataset_creation.py

```diff
@@ -50,15 +50,11 @@ def test_generate_dataset_with_collector_env(dataset_id, env_id):
     env.reset(seed=42)
 
     for episode in range(num_episodes):
-        terminated = False
-        truncated = False
-        while not terminated and not truncated:
+        done = False
+        while not done:
             action = env.action_space.sample()  # User-defined policy function
             _, _, terminated, truncated, _ = env.step(action)
-            if terminated or truncated:
-                assert not env._buffer[-1]
```

Comment:
why is it safe to remove these asserts?

Reply:
I changed the internal structure and the assert is not true anymore (when resetted, the buffer is not void, but immediately has the episode id)
Anyway, it wasn't a meaningful test (it doesn't test any wanted feature)